### PR TITLE
Richer markdown example in the playground

### PR DESCRIPTION
**Pull request by lipis**
Mostly for the tables :)

[<sub><sub></sub></sub> Preview playground link](https://deploy-preview-4176--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEIA0IIAcYEtoDOyoAhgE5kQDuACuQkSiQG4S4Am6IJBMyAZiQA2BOBgBGZEmADWcGAGUs03FADmyGGQCuYkOwhgBw0RlWiyMGlLUBbEsZF6AVgQAeAISmz5CkrbgAGVU4R1NMbRgsSIAmML1lMgtkEHsyGQMqKC4sMlUYAHUOGAALZAAOAAYMXIhRAqksFNy4C2ZQjDI4AEdtXC7rEjsHJEEnDFFbXE0dPQJVNSE4AEVtCHh4jBgScSL2UuQYralcIQWAYQhbexSoaA6QbVEAFR3GMdEAX0+gA)

### CODE REVIEW

File: website/static/playground.js [hunk 1]
```diff
@@ -496,18 +496,27 @@ function getExample(parser) {
...
+        "- - - - - - - - - - - - - - -",
+        "",
         "+ List",
         " + with a [link] (/to/somewhere)",
-        "+ and [another one]",
-        "",
         "",
-        "  [another one]:  http://example.com 'Example title'"
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+        "Curabitur consectetur maximus risus, sed maximus tellus tincidunt et."
```
Comment (lipis) [inline website/static/playground.js]: To include an example with the only option that we have specific for Markdown.

File: website/static/playground.js [hunk 2]
```diff
@@ -496,18 +496,27 @@ function getExample(parser) {
...
+        "Jane Roe|JFK|314",
+        "",
+        "- - - - - - - - - - - - - - -",
+        "",
         "+ List",
         " + with a [link] (/to/somewhere)",
-        "+ and [another one]",
-        "",
         "",
-        "  [another one]:  http://example.com 'Example title'"
```
Comment (ikatyang) [inline website/static/playground.js]: This one is the example for `--single-quote`, we shouldn't remove it.

Comment (lipis) [reply website/static/playground.js]: Right!

**Comment by lydell** [comment]
Nice!

**Pull request merged**